app/views/devise/registrations/new.html.erb

```diff
@@ -1,31 +1,37 @@
+<section class="main-container layout-centered">
 <h2>Sign up</h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: {class:'devise-form'}) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
   <div class="field">
-    <%= f.text_field :name, autofocus: true, placeholder: 'Name', class:'frm-control' %>
+    <%= f.label :"Full Name" %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
   </div>
 
   <div class="field">
-    <%= f.email_field :email, autofocus: true, placeholder: 'Email', autocomplete: "email", class: 'frm-control' %>
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
 
   <div class="field">
-    <%= f.password_field :password, class: 'frm-control', autocomplete: "new-password", placeholder: "Password #{@minimum_password_length} characters minimum" %>
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
   </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if add some inputs filed in your form such as for `user_bio`, `user_photo`,... so the `User` will be registered with all informations.

Reply:
Great, I will work on that. Thank you !!